### PR TITLE
Fix/update libraries

### PR DESCRIPTION
**Pull request by mariobodemann**
This PR will update 
- kotlin to 1.0.1
- okhttp to 3.2.0
- retrofit to 2.0.1 

isn't that awesome?

### CODE REVIEW

File: src/main/java/com/contentful/java/cma/CMACallback.java [hunk 1]
```diff
@@ -16,22 +16,20 @@
 
 package com.contentful.java.cma;
 
-import retrofit.RetrofitError;
-
 /**
  * Callback to be used with any of the {@code CMAClient} asynchronous methods.
- *
+ * <p>
```
Comment (neonichu) [inline src/main/java/com/contentful/java/cma/CMACallback.java]: Don't we have to close the paragraphs as well? 🤔

File: src/main/java/com/contentful/java/cma/interceptor/AuthorizationHeaderInterceptor.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+   *
+   * @param chain the execution chain for the request.
+   * @return the response received
+   * @throws IOException
+   */
+  @Override public Response intercept(Chain chain) throws IOException {
+    final Request request = chain.request();
+
+    return chain.proceed(request.newBuilder()
+        .addHeader(HEADER_NAME, "Bearer " + token)
```
Comment (neonichu) [inline src/main/java/com/contentful/java/cma/interceptor/AuthorizationHeaderInterceptor.java]: For consistency, either `"Bearer "` should also be a string constant.

File: src/test/kotlin/com/contentful/java/cma/AssetTests.kt [hunk 3]
```diff
@@ -16,25 +16,23 @@
 
 package com.contentful.java.cma
 
+
```
Comment (neonichu) [inline src/test/kotlin/com/contentful/java/cma/AssetTests.kt]: why the extra newline? :scream:

File: src/main/java/com/contentful/java/cma/CMACallback.java (hunk 1 above)
Comment (mariobodemann) [inline src/main/java/com/contentful/java/cma/CMACallback.java]: True, that was the autoformat. I'll remove this line and check the others if a selfclosing will be sufficient.

File: src/test/kotlin/com/contentful/java/cma/AssetTests.kt (hunk 3 above)
Comment (mariobodemann) [inline src/test/kotlin/com/contentful/java/cma/AssetTests.kt]: :+1: Good question. Just checking your eyes?

File: src/main/java/com/contentful/java/cma/interceptor/AuthorizationHeaderInterceptor.java (hunk 2 above)
Comment (mariobodemann) [inline src/main/java/com/contentful/java/cma/interceptor/AuthorizationHeaderInterceptor.java]: Good catch. :+1:

**Comment by neonichu** [comment]
:shipit:

**Pull request merged**